wikicurricula-uy/wikicurricula-boilerplate
# Fixes #28  - Add tick marks to the x axis in the main visualization page

Fixes #28 

What changes did - 
Added the X-axis which changes date and tick size based on the selected subject


Steps to test - 

- Go to main index.html file , run it in browser
- See the graph bottom, X-axis displaying along with the dynamic dates  


Result Screenshot - 

![X-axis_date](https://github.com/wikicurricula-uy/wikicurricula-boilerplate/assets/97338915/a46c256c-d824-428c-ad3c-82cc3d593a5a)
